components/UI/DisplayLostConncetion.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import styled from 'styled-components';
+import { useTranslation } from 'react-i18next';
+
+import { useAuth } from '../../lib/Auth';
+import { useMatrix } from '../../lib/Matrix';
+import LoadingSpinnerInline from './LoadingSpinnerInline';
+
+const NoConnectionView = styled.div`
+  position: absolute;
+  right: var(--margin);
+  bottom: var(--margin);
+  display: grid;
+  grid-auto-flow: column;
+  grid-gap: calc(var(--margin)/2);
+  place-items: center;
+  padding: calc(var(--margin)/2) var(--margin);
+  color: white;
+  text-transform: uppercase;
+  background-color: #ff2f4f;
+  border-radius: 4px;
+
```

Comment:
@andirueckel Can you please review in terms of UI/UX?

![image](https://github.com/medienhaus/medienhaus-spaces/assets/706419/c7ee962a-ceb6-48c9-80bf-306876d925a0)